app/assets/stylesheets/resource_allocation.scss

```diff
@@ -0,0 +1,24 @@
+// Place all the styles related to the Requests controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.graphLabel {
+    text-align: center;
+    width: 150px;
```

Comment:
Same goes for this width, I guess, and the other ones in this css :)

Reply:
same as above